apiary.apib

```diff
@@ -3067,6 +3084,37 @@ Returns navigation information
         }
 
 
+## Course Certificates Status [/courses/{course_id}/certificates/status]
+
+It returns status for course certificates
+
+### Course Certificates Status [GET]
+
+Returns a JSON representation of certificate details for a specified course, also return status if we can request certificate generation for the course, possible statuses can be 'available', 'notavailable', 'generated', 'generating'
```

Comment:
Certificate status is per course per student. So we can create an API to return certificate status of learner in a particular course but there is not certificate status per course.